### PR TITLE
 Dashboard update. Update section 

### PR DESCRIPTION
**Pull request by Poseidon281**
Reopened PR to cleanup the comments at the suggestion of Lance

This PR makes a slight modification to the Dashboard so it now also displays that the panel is up to date

### CODE REVIEW

File: resources/views/filament/pages/dashboard.blade.php [hunk 1]
```diff
@@ -9,6 +8,33 @@
...
+            :header-actions="$updateActions"
+        >
+            <x-slot name="heading">{{ trans('dashboard/index.sections.intro-update-need.heading') }}</x-slot>
+
+            <p>{{ trans('dashboard/index.sections.intro-update-need.content', ['latestVersion' => $latestVersion]) }}</p>
+
+        </x-filament::section>
+    @endif
+
+    @if ($isLatest)
```
Comment (Boy132) [inline resources/views/filament/pages/dashboard.blade.php:25]:
```suggestion
    @else
```

File: lang/en/dashboard/index.php [hunk 2]
```diff
@@ -19,10 +19,14 @@
...
             'button_features' => 'Discuss Features',
         ],
-        'intro-update' => [
+        'intro-update-need' => [
             'heading' => 'Update available',
             'content' => ':latestVersion is available! Read our documentation to update your Panel.',
         ],
+        'intro-update-done' => [
+            'heading' => 'Your Panel is up to date',
+            'content' => 'You are currently using :latestVersion. No update is available',
```
Comment (Boy132) [inline lang/en/dashboard/index.php:28]:
```suggestion
            'content' => 'You are currently using :version. Your panel is up-to-date!',
```

File: lang/en/dashboard/index.php [hunk 3]
```diff
@@ -19,10 +19,14 @@
             'button_issues' => 'Create Issue',
             'button_features' => 'Discuss Features',
         ],
-        'intro-update' => [
+        'intro-update-need' => [
             'heading' => 'Update available',
             'content' => ':latestVersion is available! Read our documentation to update your Panel.',
         ],
+        'intro-update-done' => [
```
Comment (Boy132) [inline lang/en/dashboard/index.php:26]:
```suggestion
        'intro-no-update' => [
```

File: lang/en/dashboard/index.php [hunk 4]
```diff
@@ -19,10 +19,14 @@
             'button_issues' => 'Create Issue',
             'button_features' => 'Discuss Features',
         ],
-        'intro-update' => [
+        'intro-update-need' => [
```
Comment (Boy132) [inline lang/en/dashboard/index.php:22]:
```suggestion
        'intro-update-available' => [
```

File: resources/views/filament/pages/dashboard.blade.php [hunk 5]
```diff
@@ -9,6 +8,33 @@
...
+    @if (!$isLatest)
+        <x-filament::section
+            icon="tabler-info-circle"
+            icon-color="warning"
+            id="intro-update-not-ready"
+            :header-actions="$updateActions"
+        >
+            <x-slot name="heading">{{ trans('dashboard/index.sections.intro-update-need.heading') }}</x-slot>
+
+            <p>{{ trans('dashboard/index.sections.intro-update-need.content', ['latestVersion' => $latestVersion]) }}</p>
```
Comment (Boy132) [inline resources/views/filament/pages/dashboard.blade.php:20]:
```suggestion
            <x-slot name="heading">{{ trans('dashboard/index.sections.intro-update-available.heading') }}</x-slot>

            <p>{{ trans('dashboard/index.sections.intro-update-available.content', ['latestVersion' => $latestVersion]) }}</p>
```

File: resources/views/filament/pages/dashboard.blade.php [hunk 6]
```diff
@@ -9,6 +8,33 @@
...
+
+    @if ($isLatest)
+        <x-filament::section
+            icon="tabler-checkbox"
+            icon-color="success"
+            id="intro-update-ready"
+        >
+            <x-slot name="heading">{{ trans('dashboard/index.sections.intro-update-done.heading') }}</x-slot>
+
+            <p>{{ trans('dashboard/index.sections.intro-update-done.content', ['latestVersion' => $latestVersion]) }}</p>
```
Comment (Boy132) [inline resources/views/filament/pages/dashboard.blade.php:33]:
```suggestion
            <x-slot name="heading">{{ trans('dashboard/index.sections.intro-no-update.heading') }}</x-slot>

            <p>{{ trans('dashboard/index.sections.intro-no-update.content', ['version' => $version]) }}</p>
```

File: resources/views/filament/pages/dashboard.blade.php [hunk 7]
```diff
@@ -9,6 +8,33 @@
 
     <p>{{ trans('dashboard/index.expand_sections') }}</p>
 
+    @if (!$isLatest)
+        <x-filament::section
+            icon="tabler-info-circle"
+            icon-color="warning"
+            id="intro-update-not-ready"
```
Comment (Boy132) [inline resources/views/filament/pages/dashboard.blade.php:15]:
```suggestion
            id="intro-update-available"
```

File: resources/views/filament/pages/dashboard.blade.php [hunk 8]
```diff
@@ -9,6 +8,33 @@
...
+            <p>{{ trans('dashboard/index.sections.intro-update-need.content', ['latestVersion' => $latestVersion]) }}</p>
+
+        </x-filament::section>
+    @endif
+
+    @if ($isLatest)
+        <x-filament::section
+            icon="tabler-checkbox"
+            icon-color="success"
+            id="intro-update-ready"
```
Comment (Boy132) [inline resources/views/filament/pages/dashboard.blade.php:29]:
```suggestion
            id="intro-no-update"
```

**Review by Boy132 — CHANGES_REQUESTED** [review]
Looks good, just some small adjustments.

**Comment by Poseidon281** [comment]
> Looks good, just some small adjustments.

Thanks for the suggestions. I just added them

**Review by Boy132 — APPROVED** [review]
LGTM

**Pull request merged**